### PR TITLE
docs(Enterprise): How to reset forgotten groot password.

### PR DESCRIPTION
**Pull request by danielmai**
This adds a new section to the docs to explain the how-to steps to reset the groot user password.

Related: DGRAPH-761

---
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/5676)

 
 
 

Docs Preview: [Dgraph Preview](https://dgraph-6a89e9f7ab-71908.surge.sh)

### CODE REVIEW

File: wiki/content/enterprise-features/index.md [hunk 1]
```diff
@@ -740,6 +740,34 @@ mutation {
 }
 ```
 
+### Reset Groot Password
+
+If you've forgotten the password to your groot user, then you may reset the groot password (or
+the password for any user) by the following these steps. You'll need to have access to update the
```
Comment (parasssh) [inline wiki/content/enterprise-features/index.md:746]: access to what ?

File: wiki/content/enterprise-features/index.md [hunk 2]
```diff
@@ -740,6 +740,34 @@ mutation {
...
+Dgraph cluster configuration.
+
+1. Stop Dgraph Alpha. If you're running a high available cluster, then you can stop this Alpha.
+2. Turn off ACLs by removing the `--acl_hmac_secret` config flag in the Alph config. This leaves the
+   Alpha open with no ACL rules, so be sure to secure access to this Alpha, including
```
Comment (parasssh) [inline wiki/content/enterprise-features/index.md:751]: secure -> restrict

File: wiki/content/enterprise-features/index.md [hunk 3]
```diff
@@ -740,6 +740,34 @@ mutation {
...
+   stopping request traffic to this Alpha.
+3. Start Dgraph Alpha.
+4. Connect to Dgraph using Ratel and run the following upsert mutation to update the groot password
```
Comment (parasssh) [inline wiki/content/enterprise-features/index.md:754]: Connect to Dgraph Alpha

**Comment by OmarAyo** [comment]
Once Paras' comments are addressed the PR LGTM

**Pull request merged**